docs/user/README.md

```diff
@@ -58,6 +58,9 @@ Beyond basic LSP features, there are some extension commands which you can
 invoke via <kbd>Ctrl+Shift+P</kbd> or bind to a shortcut. See [./features.md](./features.md)
 for details.
 
+For updates, pull the latest changes from the master branch, run `cargo install-ra` again, and **restart** VS Code instance.
```

Comment:
For me it required a restart between uninstall of the ext and `cargo install-ra`, otherwise it caches the old ext.

Reply:
Hm, do you mean you need to manually *uninstall* the extension as well? For me, it just overwrites the old extension...

Reply:
Never needed to uninstall the extension when upgrading either. Overwrites just fine.

Reply:
Just replacing it without uninstalling, seems to have worked this time.